pkg/apis/workspaces/v1alpha1/commands.go

```diff
@@ -117,10 +117,16 @@ type ExecCommand struct {
 	// Working directory where the command should be executed
 	WorkingDir string `json:"workingDir,omitempty"`
 
-	// +optional
 	// Optional list of environment variables that have to be set
 	// before running the command
 	Env []EnvVar `json:"env,omitempty"`
+
+	// +optional
+	// Whatever the command it capable to reload itself when source code changes.
```

Comment:
> Whatever the command it capable

Did you mean: **_whether the command is capable_** ?

Reply:
yes 😇 , fixed. thx